hw/ip/aes/dv/aes_model_dpi/aes_model_dpi.c

```diff
@@ -129,7 +129,11 @@ void c_dpi_aes_crypt_message(unsigned char impl_i, unsigned char op_i,
   const unsigned char op = op_i & op_mask;
   const crypto_mode_t mode = (crypto_mode_t)(*mode_i & mode_mask);
   if (mode == kCryptoAesNone) {
+    printf(
         "ERROR: Mode kCryptoAesNone not supported by c_dpi_aes_crypt_message");
+    return;
+  }
+
```

Comment:
Ahah! It's added back! Could you squash these hunks (or just drop them) to avoid breaking the C build between the 2nd and 3rd commits?